src/ATE_spyder/ate_spyder/widgets/main_widget.py

```diff
@@ -188,7 +188,11 @@ def project_dialog_finished(self, result, project_path=None):
             # hack: as spyder automatically create an empty project even
             # before semi-ate project validation done we need to clean up
             # after canceling creating the project
-            shutil.rmtree(project_path)
+            try:
+                shutil.rmtree(project_path, True)
+            except PermissionError:
```

Comment:
this doesn't realy fix the issue.. the cleanup will not happen and the new generated folder will not be removed 

rmtree with the 'ignore_errors' flag will basicaly hide any exception so catching the exception is redundant ...

Reply:
This is due to the limited Windows file system concurrency. The only way to get around this would be either to check somehow when the folder lock is removed or to do some of parent process acquisition. We'll check which works better